### PR TITLE
[NO-TICKET] New shortcode: contact CSM or support

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
@mjang I'm setting up this no-ticket PR to add a new shortcode as per your suggestion in https://github.com/cobalthq/cobalt-product-public-docs/pull/171

### CODE REVIEW

File: content/en/Getting started/what-to-expect.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ results. Here's what you can expect:
 1. We keep the Slack channel open until you've set each finding to:
    - Accepted Risk
    - Fixed
-   If you need access to the archived Slack channel, contact your Customer Success Manager or support@cobalt.io.
+      - {{% contact-csm-support %}} if you need access to the archived Slack channel.
```
Comment (mjang-cobalt) [inline content/en/Getting started/what-to-expect.md:58]:
```suggestion

  {{% contact-csm-support %}} if you need access to the archived Slack channel.

```

I think this looks better (from my build)
Screen Shot 2022-08-30 at 7 28 18 AM

Comment (mjang-cobalt) [reply content/en/Getting started/what-to-expect.md:58]: (Yes, I had messed up the formatting earlier.)

**Review by mjang-cobalt — APPROVED** [review]
Approving with one suggested change

File: content/en/Getting started/what-to-expect.md [hunk 2]
```diff
@@ -55,7 +55,8 @@ results. Here's what you can expect:
 1. We keep the Slack channel open until you've set each finding to:
    - Accepted Risk
    - Fixed
-   If you need access to the archived Slack channel, contact your Customer Success Manager or support@cobalt.io.
+   <br>{{% contact-csm-support %}} if you need access to the archived Slack channel.
```
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/what-to-expect.md:58]: @mjang-cobalt I changed the formatting because the next ordered item appeared as `1`.

**Pull request merged**